marti_nav_msgs/msg/PlanPoint.msg

```diff
@@ -0,0 +1,19 @@
+# Position and heading for this point in the plan
```

Comment:
should this be replaced with a geometry_msgs/Pose instead of a separate definition?

Reply:
Pose has too many degrees of freedom

Reply:
are you referring to pitch and yaw?